participants.md

```diff
@@ -21,10 +21,18 @@ His materials are also available [on Github](https://github.com/bbadzioch/MTH309
 [Francesca](http://www.fbernardi.info) is currently dean's post-doctoral scholar in the Department of Mathematics at Florida State University. 
 She is starting a tenure-track position in Mathematics with affiliation in Mechanical Engineering at the Worcester Polytechnic Institute next Fall.
 
-### Ryan Cooper
-
-[Ryan](https://me.engr.uconn.edu/blog/faculty/cooper-ryan/) is assistant professor-in-residence at the University of Connecticut Department of Mechanical Engineering. 
-He is now teaching computational mechanics using Jupyter, and his course materials are available [on GitHub](https://github.uconn.edu/rcc02007/Computational_Mechanics).
+### Ryan C. Cooper
+
+[Ryan](https://ryan-c-cooper.uconn.edu) is assistant professor-in-residence at
+the University of Connecticut Department of Mechanical Engineering. He aims to
+stengthen problem-solving skills with project-based learning and numerical
+methods.  He has used Jupyter to enhance learning in Dynamics, Vibrations,
+upper-level undergraduate laboratories, and is now teaching computational
+mechanics using Jupyter. His course materials are available [on GitHub
```

Comment:
This link looks a little weird to me, did you mean `on GitHub: [Computational ...`?

Reply:
`[on GitHub](https://github.uconn.edu/rcc02007/Computational_Mechanics)` should appear as [on GitHub](https://github.uconn.edu/rcc02007/Computational_Mechanics), which seems fine to me

Reply:
I think it says `[on GitHub Computational Mechanics](...)` if I'm reading the Markdown correctly

Reply:
I had made the link 

"[on Github Computational Mechanics](https://github.uconn.edu/rcc02007/Computational_Mechanics)", 

but its fine to change it to:

"on Github: [Computational Mechanics](https://github.uconn.edu/rcc02007/Computational_Mechanics)"

I just like __big links__